### DISCUSSION TITLE
Feature Request: Vector Line Equivalent to Legend Series Aggregations

### DISCUSSION THREAD

**Discussion by hdost** [grafana/grafana#38192] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Currently Graphs offer a way to show in the legend an aggregation across points returned such as max,min, last etc. I would like to see a way to plot that value as a vector on the graph. This could be done for either a single series or all series.

**Why is this needed**:
This is useful for seeing in the graphite interval the min, max,avg It provides a localized rolling aggregation which can be useful when you're not trying to compare intervals across a range.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
